.env.example

```diff
@@ -2,6 +2,9 @@
 ETHERSCAN_API_KEY=
 POLYGON_MUMBAI_API_KEY=
 
+# RPCs
```

Comment:
there's a section at the bottom `# Endpoint URLS` where this fits better, since all of the other RPCs are there